lib/rubocop/cop/style/commented_keyword.rb

```diff
@@ -60,6 +60,8 @@ class CommentedKeyword < Base
         SUBCLASS_DEFINITION = /\A\s*class\s+(\w|::)+\s*<\s*(\w|::)+/.freeze
         METHOD_DEFINITION = /\A\s*def\s/.freeze
 
+        STEEP_REGEXP = /#\ssteep:ignore(\s|$)/.freeze
```

Comment:
Is the end of lines in a multiline context intended? If not, `\z` would be more appropriate.

```suggestion
        STEEP_REGEXP = /#\ssteep:ignore(\s|\z)/.freeze
```

Reply:
@koic 

Thanks for your suggestion!

I did not intend the end of lines in a multiline context. 
I fixed it so that only word endings apply to regular expressions.

https://github.com/rubocop/rubocop/commit/615526c3321b7f5929a50997c7a22f720add4d2b#diff-33530142aa1144a74752b8678c37483180c7f32218b5503e08997ab076823b91R62-R63